docs/release_notes/sat_2.6_release_notes.md

```diff
@@ -0,0 +1,108 @@
+# Changes in SAT 2.6
+
+The 2.6.4 version of the SAT product includes:
+
+- Version 3.24.0 of the `sat` python package and CLI.
+- Version 2.0.0-1 of the `sat-podman` wrapper script.
+- Version 1.6.1 of the `sat-install-utility` container image.
+
+## New `sat` commands
+
+No new `sat` commands were added in SAT 2.6.
+
+## Changes to `sat bootsys`
+
+- Functionality has been added to the `platform-services` and `cabinet-power`
+  stages of `sat bootsys boot` which allows SAT to automatically re-create
+  Kubernetes CronJobs which may have become stuck during shutdown, boot, or
+  reboot.
+
+- `sat bootsys boot` will more reliably determine if the `hms-discovery` cronjob
+  has been scheduled during the `cabinet-power` stage. This same functionality
+  is also used by `sat swap blade`.
+
+- SAT will use the BatchV1 Kubernetes API to manipulate CronJobs, instead of the
+  BatchV1Beta1 API as previously.
```

Comment:
I think leaving it starting with "SAT" is fine, since `sat bootsys` isn't the only SAT command which modifies CronJobs (for instance, `sat swap blade` does as well.) I personally like the wording you suggest above.

Reply:
@jack-stanek-hpe, it seems like this bullet and the one on lines 20-22 both discuss `sat bootsys` and `sat swap blade`. However, I feel like that's lost a little bit with the current wording. What do you think of splitting the commands apart for more clarity? I believe I did something similar for changes to some SAT 2.5 commands.

You can change the information above to something like the following:

```
- `sat bootsys boot` more reliably determines if the `hms-discovery` CronJob was scheduled
   during the `cabinet-power` stage.

- `sat bootsys boot` will use the BatchV1 Kubernetes API to manipulate CronJobs, instead of
   the BatchV1Beta1 API as previously.
```

Then, add something like the following to the "Other SAT Changes" section:

```
- `sat swap blade` more reliably determines if the `hms-discovery` CronJob was scheduled
  during the `cabinet-power` stage.

- `sat swap blade` will use the BatchV1 Kubernetes API to manipulate CronJobs, instead of
  the BatchV1Beta1 API as previously.
```

Reply:
Jessica's suggested wording and reorganization looks good to me, with the exception of "during the `cabinet-power` stage" in the "Other SAT Changes" section.